### DISCUSSION TITLE
Problem running Neovim on code-server Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by mintgh** [coder/code-server#5126] [Q&A]
I have Code-server running on Ubuntu 20.04, for some reasons when I installed Neovim, it said I needed to add the path to the extension executable file, now I don’t know which file should I point to, since there quite a few of .vim files in there. Also I get EACCESS error when I try to point the path to test out those .vim files, any helps would be appreciated
![IMG_0047](https://user-images.githubusercontent.com/87857426/164950356-033a4a26-cf78-429c-b16b-700d2017f9e2.jpeg)
![IMG_0048](https://user-images.githubusercontent.com/87857426/164950383-e460792a-3357-42dc-81b8-0844e96d54f3.jpeg)

**Comment by jsjoeio** [comment]
Ah yes! I had trouble with this the other day too. It wants the path to the neovim executable. I think you should be able to get this by running `which nvim` 

<img width="1337" alt="image" src="https://user-images.githubusercontent.com/3806031/165186858-b60a78be-1587-47b9-ba9b-e41bb4d17c17.png">

Then you'll need to update the value in Settings so the extension can find it.

**Reply by mintgh** [reply]
Hey thanks for helping. Using `which nvim` for me didn't work because I installed code-server using yarn and the way it saved files from code-server stuff is kinda weird(I consider myself to be a linux noob). Though I found the Nvim folder through this `PATH: /home/[$USER]/.local/share/code-server/extensions/asvetliakov.vscode-neovim-0.0.83/vim`. I then put the PATH into the setting:
![image](https://user-images.githubusercontent.com/87857426/165220965-7674c51e-f840-40be-b74f-685f44d08d63.png)

But when I refresh the webpage to reload the extension, I would get this error:
![image](https://user-images.githubusercontent.com/87857426/165221012-ea3ab858-fcf8-497b-a772-dbadecd0d49d.png)
Then I thought maybe the Vim folder was restricting access, so I ran `chmod 777` for the folder and retry but still no luck.

TLDR; 
Still not working :(

**Reply by lg-nineoneninethreenine** [reply]
What @jsjoeio meant was that you first install the real Neovim on your computer and then run `which nvim` on the CLI. NVIM is the command for Neovim btw.
Once you get the binary path, enter that in your Settings page, and restart your session. Then, enjoy!

**Reply by jsjoeio** [reply]
Yes! What @im-coder-lg said. I think we're confusing two things.

You want the path to the executable for `nvim`, not the path to the extension. Does that make sense?
